taskfiles/lint/yaml.yaml

```
@@ -0,0 +1,24 @@
+version: "3"
+
+tasks:
+  check-yaml:
+    desc: "Runs the YAML linters."
+    sources:
+      - "{{.G_LINT_VENV_CHECKSUM_FILE}}"
+      - "{{.ROOT_DIR}}/**/*.yaml"
+      - "{{.ROOT_DIR}}/**/*.yml"
+      - "{{.TASKFILE}}"
+      - exclude: "{{.ROOT_DIR}}/**/build/*"
+    dir: "{{.ROOT_DIR}}"
+    deps: ["venv"]
+    cmds:
+      - |-
+        . "{{.G_LINT_VENV_DIR}}/bin/activate"
+        find . \
+          \( -path '*/build/*' \) -prune -o \
+          \( -iname "*.yaml" -o -iname "*.yml" \) \
```

```suggestion
          \( -iname "*.yaml" \) \
```

Same as the workflow, let's not include `.yml` files.

---

I think this comment still needs to be addressed.